operating system Configuration

```diff
@@ -0,0 +1,44 @@
+Software Configuration:
+A Debian based Linux distribution known as Raspbian OS will be installed onto a Raspberry Pi to collect data over a serial interface and send collected data to a processing server via wifi network connectivity. 
+
+Minimum requirements for the Raspbian OS:
+The basic install provides only a terminal interface, in order to display an app in a GUI on the screen a X11 desktop environment running a minimal user interface called Open Box will be used.  The application portion of the kiosk will run a Chromium browser in “kiosk” mode.
+
+“Kiosk” mode of chromium will lock the user environment to a specific website (our electron app) allowing interaction via touch screen and possibly a hard wired USB Keyboard.
+
+The OS will need to be configured to automatically launch the web browser after a reboot and require no user interaction, making the whole system run like a simple appliance serving just the ELO app
+
+* Install Raspbian light to a SD card
+* Sign in with (default user is ‘pi’ , and the password is ‘raspberry’) then run:
+sudo raspi-config
+    * 	•	Change the password
+        * User: Pi    
+        * Password: ELOg0@l
+    * Configure network automatically to join Solstice wifi network
+    * Enable SSH access (for remote adjustments and maintenance)
+    * Enable boot to desktop without logging in
+    * Adjust GPU memory if needed
+    * Finish and exit 
+* Update the system package manager run:
```

Comment:
The format is a little off with these terminal commands. Can we clean these up?

Reply:
ok I removed the unwanted bullet point and changed the * to a - for user: pi and Password ELOg0@l
I also fixed the username caseing to lower case